--- ./tmp_original_source/server/channels/app/app.go	2026-01-05 09:02:00.000000000 +0100
+++ ./changed_files/server/channels/app/app.go	2026-01-16 15:58:24.859220257 +0100
@@ -9,6 +9,9 @@
 	"strconv"
 	"time"
 
+    "github.com/coreos/go-oidc/v3/oidc"
+    "golang.org/x/oauth2"
+
 	"github.com/mattermost/mattermost/server/public/model"
 	"github.com/mattermost/mattermost/server/public/shared/httpservice"
 	"github.com/mattermost/mattermost/server/public/shared/mlog"
@@ -25,6 +28,11 @@
 // and its only purpose is to provide business logic to Server via its methods.
 type App struct {
 	ch *Channels
+
+    // OIDC fields
+    oidcProvider *oidc.Provider
+    oidcConfig   *oauth2.Config
+    oidcVerifier *oidc.IDTokenVerifier
 }
 
 func New(options ...AppOption) *App {
