--- ./tmp_original_source/webapp/channels/src/components/header_footer_route/header.tsx	2026-01-05 09:02:00.000000000 +0100
+++ ./changed_files/webapp/channels/src/components/header_footer_route/header.tsx	2026-01-19 14:45:35.757451590 +0100
@@ -24,17 +24,13 @@
     const {SiteName} = useSelector(getConfig);
     const license = useSelector(getLicense);
 
-    const ariaLabel = SiteName || 'Mattermost';
+    const ariaLabel = SiteName || 'Mattersbearly';
 
     let freeBanner = null;
-    if (license.IsLicensed === 'false') {
-        freeBanner = <><Logo/><span className='freeBadge'>{'TEAM EDITION'}</span></>;
-    } else if (license.SkuShortName === LicenseSkus.Entry) {
-        freeBanner = <><Logo/><span className='freeBadge'>{'ENTRY EDITION'}</span></>;
-    }
+    freeBanner = <><Logo/></>;
 
     let title: React.ReactNode = SiteName;
-    if (title === 'Mattermost') {
+    if (title === 'Mattersbearly') {
         if (freeBanner) {
             title = '';
         } else {
