--- ./tmp_original_source/webapp/channels/src/components/global_header/left_controls/product_menu/product_branding_team_edition/product_branding_free_edition.tsx	2026-01-05 09:02:00.000000000 +0100
+++ ./changed_files/webapp/channels/src/components/global_header/left_controls/product_menu/product_branding_team_edition/product_branding_free_edition.tsx	2026-01-17 21:42:46.015505709 +0100
@@ -44,20 +44,12 @@
 const ProductBrandingFreeEdition = (): JSX.Element => {
     const license = useSelector(getLicense);
 
-    let badgeText = '';
-    if (license?.SkuShortName === LicenseSkus.Entry) {
-        badgeText = 'ENTRY EDITION';
-    } else if (license?.IsLicensed === 'false') {
-        badgeText = 'TEAM EDITION';
-    }
-
     return (
         <ProductBrandingFreeEditionContainer tabIndex={-1}>
             <StyledLogo
                 width={116}
                 height={20}
             />
-            <Badge>{badgeText}</Badge>
         </ProductBrandingFreeEditionContainer>
     );
 };
