--- ./tmp_original_source/webapp/channels/src/components/widgets/menu/menu_items/menu_start_trial.tsx	2026-01-05 09:02:00.000000000 +0100
+++ ./changed_files/webapp/channels/src/components/widgets/menu/menu_items/menu_start_trial.tsx	2026-01-17 22:53:18.511725890 +0100
@@ -77,13 +77,13 @@
                     formatMessage(
                         {
                             id: 'navbar_dropdown.versionText',
-                            defaultMessage: 'This is the free <link>unsupported</link> edition of Mattermost.',
+                            defaultMessage: 'This Mattersbearly.',
                         },
                         {
                             link: (msg: React.ReactNode) => (
                                 <ExternalLink
                                     location='menu_start_trial.unsupported-link'
-                                    href={LicenseLinks.UNSUPPORTED}
+                                    href='https://github.com/davrot/mattersbearly'
                                 >
                                     {msg}
                                 </ExternalLink>
@@ -96,4 +96,4 @@
     );
 };
 
-export default MenuStartTrial;
+export default MenuStartTrial;
\ No newline at end of file
